extensions/markdown-language-features/src/languageFeatures/copyFiles/dropIntoEditor.ts

```diff
@@ -84,14 +89,23 @@ export function createUriListSnippet(document: vscode.TextDocument, uris: readon
 
 		const ext = URI.Utils.extname(uri).toLowerCase().replace('.', '');
 		const insertAsImage = typeof options?.insertAsImage === 'undefined' ? imageFileExtensions.has(ext) : !!options.insertAsImage;
-
-		snippet.appendText(insertAsImage ? '![' : '[');
-
-		const placeholderText = options?.placeholderText ?? (insertAsImage ? 'Alt text' : 'label');
-		const placeholderIndex = typeof options?.placeholderStartIndex !== 'undefined' ? options?.placeholderStartIndex + i : undefined;
-		snippet.appendPlaceholder(placeholderText, placeholderIndex);
-
-		snippet.appendText(`](${mdPath})`);
+		const insertAsVideo = videoFileExtensions.has(ext);
+
+		if (insertAsVideo) {
+			snippet.appendText(`<video src="${mdPath}" placeholder="${mdPath}" autoplay loop controls muted title="`);
+			snippet.appendPlaceholder('Title');
+			snippet.appendText(`">`);
+			snippet.appendPlaceholder('\n\t\tSorry, your browser doesn\'t support HTML 5 video.\n');
```

Comment:
Basically all browsers support the video element now (but not all formats) so let's remove the fallback text. This lets us avoid having to worry about localizing the text too